stable/metricbeat/README.md

```diff
@@ -0,0 +1,69 @@
+# Metricbeat
+
+[metricbeat](https://www.elastic.co/guide/en/beats/metricbeat/current/index.html) is used to ship Kubernetes and host metrics to multiple outputs.
+
+## Prerequisites
+
+- Kubernetes 1.9+
+
+## Installint the Chart
+
+To install the chart with the release name `my-release`:
+
+```bash
+$ helm install --name my-release stable/metricbeat
+```
+
+> **Tip**: List all releases using `helm list`
+
+## Uninstalling the Chart
+
+To uninstall/delete the `my-release` deployment:
+
+```bash
+$ helm delete my-release
+```
+
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## Configuration
+
+The following table lists the configurable parameters of the metricbeat chart and their default values.
+
+|             Parameter               |            Description             |                    Default                |
+|-------------------------------------|------------------------------------|-------------------------------------------|
+| `image.repository`                  | The image repository to pull from  | `docker.elastic.co/beats/metricbeat`       |
+| `image.tag`                         | The image tag to pull              | `6.2.4`                                   |
+| `image.pullPolicy`                  | Image pull policy                  | `IfNotPresent`                            |
+| `rbac.create`                       | If true, create & use RBAC resources | `true`                                  |
+| `serviceAccount.create`             | If true, create & use ServiceAccount | `true`       |
```

Comment:
serviceAccount.name should be configurable, see: https://github.com/helm/helm/blob/master/docs/chart_best_practices/rbac.md - add it to the docs too